src/containers/pages/PlanAssignment/helpers/JurisdictionTableView/tests/__snapshots__/index.test.tsx.snap

```diff
@@ -133,43 +133,9 @@ Array [
 ]
 `;
 
-exports[`PlanAssignment/JurisdictionTable works as expected: JurisdictionCell 1`] = `
-<span
-  key="xyz0d71d-0410-45d3-8305-a9f092a150b8-span"
->
-  ra_ksh_3
-</span>
-`;
-
-exports[`PlanAssignment/JurisdictionTable works as expected: Table headers 1`] = `
-<thead
-  className="thead-plan-orgs"
->
-  <tr>
-    <th
-      style={
-        Object {
-          "width": "25%",
-        }
-      }
-    >
-      Name
-    </th>
-    <th
-      style={
-        Object {
-          "width": "25%",
-        }
-      }
-    >
-      Team Assignment
-    </th>
-    <th />
-  </tr>
-</thead>
-`;
+exports[`PlanAssignment/JurisdictionTableView works as expected: Table headers 1`] = `null`;
```

Comment:
is this intentionally null, g

Reply:
Table headers were moved to a different component, addressed via https://github.com/onaio/reveal-frontend/pull/1194/commits/4fe94d99db40c4867eb98b468f296f8244b5f286